### PR TITLE
vm-1060-multi-fulfillment

### PR DESCRIPTION
**Pull request by vladimir-buravlev**
# Project Task

Closes #

### CODE REVIEW

File: src/modules/offers/pages/offers-details.vue [hunk 1]
```diff
@@ -403,17 +448,19 @@ import {
 
 export default defineComponent({
   url: "offer",
+  components: { VcRow },
```
Comment (asvishnyakov) [inline src/modules/offers/pages/offers-details.vue:451]: This is redundant

File: src/modules/settings/pages/fulfillment-centers/fulfillment-center-details.vue [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+    default: undefined,
+  },
+
+  options: {
+    type: Object,
+    default: () => ({}),
+  },
+});
+
+const emit = defineEmits(["page:close", "parent:call"]);
```
Comment (asvishnyakov) [inline src/modules/settings/pages/fulfillment-centers/fulfillment-center-details.vue:95]: It's not something you should fix now, but in future it's better to use typed props and emits: https://github.com/VirtoCommerce/vendor-portal/blob/c80886633faf64b9a8bcdb2d01f5705d1fde580a/src/modules/rating/pages/review-details.vue#L102-L122

File: src/modules/settings/pages/fulfillment-centers/fulfillment-centers-list.vue [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+import { defineComponent, ref, computed, watch, onMounted } from "vue";
+import {
+  UserPermissions,
+  IBladeToolbar,
+  ITableColumns,
+} from "../../../../types";
+
+export default defineComponent({
+  url: "fulfillment-centers-list",
+  permissions: [UserPermissions.SellerUsersManage],
```
Comment (asvishnyakov) [inline src/modules/settings/pages/fulfillment-centers/fulfillment-centers-list.vue:57]: Are you sure we need to check this permission here? I think https://github.com/VirtoCommerce/vc-module-marketplace-vendor/blob/cae2e3d95022f3825083a148d6a24cd1eff3dc00/src/VirtoCommerce.MarketplaceVendorModule.Core/ModuleConstants.cs#L24 is candidate to use here

File: src/modules/offers/pages/offers-details.vue (hunk 1 above)
Comment (vladimir-buravlev) [reply src/modules/offers/pages/offers-details.vue]: yes

File: src/modules/settings/pages/fulfillment-centers/fulfillment-centers-list.vue (hunk 3 above)
Comment (vladimir-buravlev) [reply src/modules/settings/pages/fulfillment-centers/fulfillment-centers-list.vue]: Done.

File: src/modules/settings/pages/fulfillment-centers/fulfillment-center-details.vue (hunk 2 above)
Comment (vladimir-buravlev) [reply src/modules/settings/pages/fulfillment-centers/fulfillment-center-details.vue:95]: Thank you!

**Review by asvishnyakov — APPROVED** [review]

**Review by maksimzinchuk — APPROVED** [review]

**Pull request merged**